src/gtest/test_mempool.cpp

```diff
@@ -203,7 +203,7 @@ TEST_F(MempoolTest, TxInputLimit) {
 TEST_F(MempoolTest, OverwinterNotActiveYet) {
     // UpdateNetworkUpgradeParameters(Consensus::UPGRADE_OVERWINTER, Consensus::NetworkUpgrade::NO_ACTIVATION_HEIGHT);
 
-    CTxMemPool pool(::minRelayTxFee);
+    std::unique_ptr<CTxMemPool> pool(::minRelayTxFee, DEFAULT_MAX_MEMPOOL_SIZE_MB * 1000000);
```

Comment:
Ridicolously minor comment (line 216 below):
```suggestion
 EXPECT_FALSE(AcceptTxToMemoryPool(*pool, state1, tx1, LimitFreeFlag::OFF, &missingInputs, RejectAbsurdFeeFlag::OFF, DisconnectingFlag::OFF,
                                      MempoolProofVerificationFlag::SYNC));
```
(btw, it would not compile anyway)

Reply:
Done